pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
```

Comment:
Let's exclude migrations for now

Reply:
I've excluded them globally from being touched by any pre-commit hook